src/server/auth/cmds/cmds_test.go

```diff
@@ -30,12 +31,17 @@ func activateAuth(t *testing.T) {
 	require.NoError(t, err)
 	if string(out) != "ACTIVE" {
 		cmd = tu.Cmd("pachctl", "enterprise", "activate", tu.GetTestEnterpriseCode())
-		require.NoError(t, cmd.Run())
+		// run cmd in retry loop--sometimes s3 reads flake
```

Comment:
I'm not sure where an s3 read happens in here. Per discussion it sounds like you're going to move this backoff someplace else.